src/Tribe/RSVP.php

```diff
@@ -2611,8 +2611,12 @@ public function parse_attendee_details() {
 
 		$first_attendee = [];
 
-		if ( ! empty( $_POST['tribe_tickets']['attendees'] ) ) {
-			$first_attendee = $_POST['tribe_tickets']['attendees'][0];
+		if ( ! empty( $_POST['tribe_tickets'] ) ) {
+			$first_ticket = current( $_POST['tribe_tickets'] );
```

Comment:
* :warning: Detected usage of a non-sanitized input variable: $_POST['tribe_tickets']
* :no_entry_sign: Processing form data without nonce verification.

<sub>hash: 95d5f9c09c4a8fcfc433ad46c8d185ae</sub>